### PR TITLE
Mentors backend 1

### PR DESCRIPTION
**Pull request by Sami-2000**
Create basic mentor class and mentor servlet to send a list of hard-coded, simplified (only name and description) mentors.

### CODE REVIEW

**Comment by Richie78321** [comment]
nit: Would you be able to update the directories of these servlet files so they comply with [Java's directory convention for packages](https://docs.oracle.com/javase/tutorial/java/package/managingfiles.html)? For reference, this is also the structure used in our portfolios. Should be `webapp/java/com/google/opensesame/`.

File: src/main/java/MentorObject.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+public class MentorObject {
+  private String name;
+  private String description;
```
Comment (Richie78321) [inline src/main/java/MentorObject.java:5]: We should discuss how we want to structure our data before we commit to this, but this definitely works for now. Take a look at what we have in the data doc we made yesterday and feel free to add your thoughts.

Comment (Richie78321) [reply src/main/java/MentorObject.java:5]: For example, I'm wondering if the `name` property should be part of a more general User class, as both mentors and mentees will have names.

File: src/main/java/MentorsServlet.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
```
Comment (Richie78321) [inline src/main/java/MentorsServlet.java:1]: Same as above comment
```suggestion
package com.google.opensesame.servlets;
```

File: src/main/java/MentorsServlet.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+@WebServlet("/mentors")
+public class MentorsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<MentorObject> mentors = new ArrayList<MentorObject>();
+    mentors.add(new MentorObject("Obi", "Obi is awesome."));
+    mentors.add(new MentorObject("Sami", "Sami is fun."));
+    mentors.add(new MentorObject("Richi", "Richi is cool."));
```
Comment (Richie78321) [inline src/main/java/MentorsServlet.java:19]: I see we're committing to Richi lol

**Comment by Sami-2000** [comment]
Thanks for pointing out the com/google structuring! Are you sure the java folder goes in the webapp folder though? (totally cool if u r, but I recall that java was directly in main for the portfolio project)

File: src/main/java/MentorObject.java (hunk 1 above)
Comment (Sami-2000) [reply src/main/java/MentorObject.java:5]: For sure! Let's touch base during sunk.

**Comment by Richie78321** [comment]
> Thanks for pointing out the com/google structuring! Are you sure the java folder goes in the webapp folder though? (totally cool if u r, but I recall that java was directly in main for the portfolio project)

Oh yeah my bad you're right about that. Crap sorry.

**Pull request merged**